guide/10-mapping-and-visualization-alt/part7_utility_functions_for_the_map_widget.ipynb

```diff
@@ -0,0 +1,357 @@
+{
```

Comment:
<p>Since the JupyterLab part is not ready yet, let us rename part 7 to part 6, so it flows with the rest of the parts. Please rename title and filename accordingly</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>